### PR TITLE
HW-1.1

### PR DESCRIPTION
**Pull request by serg-ksv**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,9 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1 && result2 == bet2) ? 2 :
+                ((result1 > result2 && bet1 > bet2)
+                    || (result1 < result2 && bet1 < bet2)
+                    || (result1 == result2 && bet1 == bet2)) ? 1 : 0;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: '?' shoud be on a new line. Keep the block of code in line.

Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
        return (result1 == bet1 && result2 == bet2) 
        ?  2 : ((result1 > result2 && bet1 > bet2)
            || (result1 < result2 && bet1 < bet2)
            || (result1 == result2 && bet1 == bet2)) 
            ? 1 : 0;
```

Comment (serg-ksv) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

Comment (serg-ksv) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**